packages/slate-editor/src/modules/editor/Editor.tsx

```diff
@@ -141,8 +141,45 @@ export const Editor = forwardRef<EditorRef, EditorProps>((props, forwardedRef) =
         [setFloatingAddMenuOpen],
     );
 
-    const extensions = Array.from(
-        getEnabledExtensions({
+    const extensions = useMemo(
```

Comment:
We are now memoizing props provided in [`StoryEditor`](https://github.com/prezly/prezly/pull/15006) so we can memoize the extensions here as well.

This increases the performance a lot because the editor no longer has to re-render as much.